usr.bin/fstat/fuser.c

```diff
@@ -199,17 +197,8 @@ do_fuser(int argc, char *argv[])
 			flags |= KFLAG;
 			break;
 		case 's':
-			if (isdigit(*optarg)) {
-				sig = strtol(optarg, &ep, 10);
-				if (*ep != '\0' || sig < 0 || sig >= sys_nsig)
-					errx(EX_USAGE, "illegal signal number" ": %s",
-					    optarg);
-			} else {
-				sig = str2sig(optarg);
-				if (sig < 0)
-					errx(EX_USAGE, "illegal signal name: "
-					    "%s", optarg);
-			}
+			if (str2sig(optarg, &sig) < 0)
```

Comment:
I prefer `!= 0` but that's entirely a matter of taste (and of consistency with surrounding code).